### ISSUE TITLE
Do you know if the microphone will ever work?

### ISSUE THREAD

**Issue by Gituserflow** [Ellpeck/ObsidianCustomFrames#135]
Hello, I would like to know if it is possible to activate options so that the microphone works with this plugin.
![image_2024-09-24_225225067](https://github.com/user-attachments/assets/6c27fc7d-2368-4890-8561-c9e7f30db3bf)

**Comment by pyjoku** [comment]
Yes, I have the same question. I want to use audiopen for my dictation. I always get the popup "you must allow your microphone" on mac. Obsidian itself can record audio.
I used Desktop and normal iframe settings both didn't work.
Hope this feature would be possible.

**Comment by pyjoku** [comment]
https://stackoverflow.com/questions/72024011/electron-app-not-asking-for-camera-and-microphone-permission-on-macos-monterey

I found this about electron and mac. Maybe that's what we need.

**Comment by Ellpeck** [comment]
Hi! Does using the microphone work
- in Canvas embeds or
- in the new Web viewer core plugin?
If it doesn't work in either of those, it's likely an issue with the way that Obsidian's permissions are setup, and there might not be anything I can do about it. If it works in one of those, we might be able to get it working in Custom Frames as well!

**Comment by Ellpeck** [comment]
I briefly talked to licat about this, and it seems that all additional permissions (like location & microphone & whatnot) are currently blocked by default, and would potentially need a custom permission prompt to be implemented. This means that I think this is out of my hands for the time being, and we'll have to see what the Web viewer core plugin brings in the future that we can integrate with for custom permissions!
